CHANGELOG.md

```diff
@@ -43,7 +43,12 @@ per cluster.
   Note that if the scheduler is AWS Batch, the root volumes of the compute nodes cannot be encrypted by ParallelCluster.
 - Change tags prefix from `aws-parallelcluster-` to `parallelcluster:`.
 - Enable EFA for a compute resource by default if the instance type supports EFA.
-- Use tags prefix `parallelcluster:`.
+- Rename tags (the following tags are crucial for ParallelCluster scaling logic): 
```

Comment:
Rename tags. Note: the following tags...

Reply:
Done